toolkit/tools/scheduler/scheduler.go

```diff
@@ -387,6 +388,19 @@ func buildAllNodes(stopOnFailure, isGraphOptimized, canUseCache bool, packagesTo
 					}
 				}
 
+				if res.Node.Type == pkggraph.TypeBuild && res.WasDelta {
+					logger.Log.Tracef("This is a delta result, update the graph with the new delta files for '%v'.", res.Node)
+					// We will need to update the graph with paths to any delta files that were actually rebuilt.
+					err = setAssociatedDeltaPaths(res, res.BuiltFiles, pkgGraph, graphMutex)
+					if err != nil {
+						// Failures to manipulate the graph are fatal.
+						err = fmt.Errorf("error setting delta paths for ancillary nodes: %w", err)
+						logger.Log.Error(err)
```

Comment:
In the other few places we started using this approach with `%w` for errors (`PruneUnknownPackages` in `buildlist.go` for instance), we let the top caller print them to avoid unnecessary multiple error print-outs with the same message. I'd stick to that approach, since we both go the messages out, while also didn't add any noise from duplication.
```suggestion
						err = fmt.Errorf("error setting delta paths for ancillary nodes:\n%w", err)
```

Note: I've also added `\n` to the message.

Reply:
Seems reasonable. I expect this log is just from my debugging as well.